pkg/utils.py

```diff
@@ -262,7 +262,17 @@ def get_version(self, uuids: List[str], match: Matcher) -> dict:
             match (Matcher): the fmatch instance
         """
         test = match.get_results("", uuids, {})
-        return {run[self.uuid_field]: run["ocpVersion"] for run in test}
+        if len(test) == 0:
+            return {}
+        if "ocpVersion" in test[0]:
+            return {run[self.uuid_field]: run["ocpVersion"] for run in test}
+
+        result = {}
+        for uuid in uuids :
+            test = match.get_results("", [uuid], {})
+            result[uuid] = test[0]["metadata"]["ocpVersion"]
```

Comment:
Is `["metadata"]["ocpVersion"]` correct place to get hte `ocpVersion`? Doesn't align with previous logic.